tests/cxx/unit_tests/ghostfragment/testing/weighted_distance.hpp

```diff
@@ -0,0 +1,104 @@
+#pragma once
+#include "hydrocarbon/hydrocarbon.h"
+#include <ghostfragment/capping/capping.hpp>
+#include <ghostfragment/connectivity/connectivity.hpp>
+#include <ghostfragment/property_types/capped.hpp>
+#include <ghostfragment/property_types/connectivity_table.hpp>
+#include <ghostfragment/property_types/broken_bonds.hpp>
+#include <catch2/catch.hpp>
+#include <iostream>
+
+namespace testing{
+using bonds_pt = ghostfragment::pt::BrokenBonds;
+using traits_t   = ghostfragment::pt::CappedTraits;
+using result_type = traits_t::result_type;
+using set_type = result_type::value_type;
+using atom_type  = set_type::atom_type;
+
+inline auto caps_methane_one() {
+    result_type caps;
+
+    set_type cap_set1;
+
+    caps.emplace_back(std::move(cap_set1));
+
+    return caps;
+}
+
+inline auto caps_ethane_one() {
+    result_type caps;
+
+    set_type cap_set1;
+    cap_set1.add_cap(0, 1, atom_type("H", 1ul, 1837.289, 1.68228, 1.18892, 0.0));
```

Comment:
How are these coordinates computed? I don't think they are correct.

Reply:
That is from Isaac's Mathematica notebook. It uses only the basic bond length. I've been ignoring it because I thought we were just going to use VMD to visually check everything.

Reply:
VMD is fine, but you then need to update the correct answer based on the coordinates you fed into VMD.

Reply:
The correct coordinates have been entered, and now all tests pass. I'll commit the new stuff, then the PR will be ready.